.github/workflows/tokenomics.yml

```diff
@@ -21,27 +21,12 @@ jobs:
       - uses: actions/checkout@v4
         with:
           fetch-depth: 0
-      - uses: actions/setup-node@v3
+      - uses: actions/setup-node@v4
         with:
-          node-version: 18
+          node-version: '20.x'
```

Comment:
updated node version here as well as we decided to move tokenomics to 20 (though we didn't do that in Vercel, neither for current tokenomics, nor for monorepo, do you want me to do that? @mohandast52 )

Reply:
I'd start with the monorepo as it's not in prod :D